lib/util.js

```diff
@@ -369,6 +369,30 @@ exports.getVideoDescription = (html) => {
     new Entities().decode(exports.stripHTML(description[1])) : '';
 };
 
+/**
+ * Get video media (extra information) from html
+ * 
+ * @param {String} body
+ * @return {Object}
+ */
+exports.getVideoMedia = (body) => {
+  let mediainfo = exports.between(body,
+    '<div id="watch-description-extras">',
+    '<div id="watch-discussion" class="branded-page-box yt-card">');
+  if (mediainfo === '') {
+    return {};
+  }
+  mediainfo = new Entities().decode(mediainfo);
+
+  const regexp = /<h4 class="title">([\s\S]*?)<\/h4>[\s\S]*?<ul.*?class=".*?watch-info-tag-list\">[\s\S]*?<li>([\s\S]*?)<\/li>/g;
```

Comment:
No need to escape the `"` character in `watch-info-tag-list\"`

Reply:
Oops, sorry